arch/arm/src/stm32l4/stm32l4xrxx_rcc.c

```diff
@@ -789,15 +785,6 @@ static void stm32l4_stdclockconfig(void)
       regval |= STM32L4_RCC_CFGR_PPRE1;
       putreg32(regval, STM32L4_RCC_CFGR);
 
-#ifdef CONFIG_STM32L4_RTC_HSECLOCK
```

Comment:
This option still needed to select RTC clock source.  The lines are proposed to delete because L4 does not have RTCPRE register.  The HSE source prescaler is fixed to HSE/32.